### PR TITLE
Add Lunar Keyboard

### PR DESCRIPTION
**Pull request by ai03-2725**
Adds the Lunar by Switchplate Peripherals.

The two supplied keymaps by default:
- Default: The basic keymap.
- Via: A variant specifically for VIA configurator compatibility. [VIA side functionality is ready.](https://github.com/olivia/via-config/pull/43)

## Types of changes

- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by this PR

None

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by Wilba6582** [comment]
@drashna approved by me

File: keyboards/ai03/lunar/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+ */
+#include QMK_KEYBOARD_H
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+  MANUAL = SAFE_RANGE,
+  SWPLURL
+};
+
+#define _______ KC_TRNS
```
Comment (mechmerlin) [inline keyboards/ai03/lunar/keymaps/default/keymap.c]: This is actually included when you `#include QMK_KEYBOARD_H`

```suggestion

```

File: keyboards/ai03/lunar/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define _______ KC_TRNS
```
Comment (mechmerlin) [inline keyboards/ai03/lunar/keymaps/via/keymap.c]: Same comment as above
```suggestion

```

File: keyboards/ai03/lunar/readme.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Lunar
+
+![Photo](https://i.imgur.com/hYoW8IJ.jpg)
+
+A 65% keyboard made specifically for Alps SKCM/L switches and AEK/M0116 keycaps.
+
+Keyboard Maintainer: [ai03](https://github.com/ai03-2725)  
+Hardware Supported: Official Lunar PCB
```
Comment (mechmerlin) [inline keyboards/ai03/lunar/readme.md]: You need a few spaces here to get markdown to render properly

```suggestion
Hardware Supported: Official Lunar PCB  
```

**Pull request merged**